src/mozilla_taskgraph/util/signed_artifacts.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+def get_signed_artifacts(input, formats, behavior=None):
+    """
+    Get the list of signed artifacts for the given input and formats.
+    """
+    artifacts = set()
+    if input.endswith(".dmg"):
+        artifacts.add(input.replace(".dmg", ".tar.gz"))
```

Comment:
I see this is not pulling in the `mac_sign` special case from https://searchfox.org/mozilla-central/rev/04a2c5317c0af560ed1689304498416c9c6c485a/taskcluster/gecko_taskgraph/util/signed_artifacts.py#176-177. This should be fine, because that appears to be dead code.

Reply:
Yeah, I think so.. or if it isn't dead code, I was planning to have Gecko handle that edge case elsewhere.